### ISSUE TITLE
[accessibility] Interactive markers should have a default accessible name

### ISSUE THREAD

**Issue by Malvoz** [maplibre/maplibre-gl-js#355]
Interactive markers (e.g. markers that bind to popups) should have a default [accessible name](https://www.w3.org/TR/accname/#dfn-accessible-name), e.g. `aria-label="Marker"` if not specified.

Related WCAG 2.1 Success Criterion: [4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value) (this issue is to address the <q>Name</q> part of the Success Criterion for interactive markers).

<hr>

Furthermore, ideally developers should ensure that each marker has a unique and descriptive name, such that the user can know what the marker is visually highlighting (in the case of markers binding to popups it'd also allow <span title="Assistive technology">AT</span> users to know what it's for before opening the popup). For example, the marker in https://maplibre.org/maplibre-gl-js-docs/example/set-popup/ would ideally be named "Washington Monument" or similar. Perhaps developers can be warned if markers are not uniquely named?

<hr>

(The same issue was also reported with MapBox: https://github.com/mapbox/mapbox-gl-js/issues/11030.)
